modules/compression/docs/api-reference/lz4-inflate-transform.md

```diff
@@ -0,0 +1,13 @@
+# LZ4InflateTransform
+
+<p class="badges">
+  <img src="https://img.shields.io/badge/From-v2.3-blue.svg?style=flat-square" alt="From-v2.3" /> 
+</p>
+
+## Static Methods
+
+#### `LZ4InflateTransform.deflate(data: ArrayBuffer, options?: object): Promise<ArrayBuffer>`
```

Comment:
Even with the `InflateTransform`, the method is still `deflate`, not `inflate`?

Reply:
Good catch